### PR TITLE
Doc manually gen key/cert multi-process container

### LINKED ISSUE #389: Write multi-process container docs for manually generating keys/certs
We should write multi-process container docs for manually generating / import keys/certs.

### PR DESCRIPTION
**Pull request by git-hyagi**
closes #389

### CODE REVIEW

File: docs/certificates.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+MIIDTTCCAjWgAwIBAgIUZFS+5+hhWrM270+X+k8vpfwIQL8wDQYJKoZIhvcNAQEL
+BQAwGzEZMBcGA1UEAwwQcHVscC5leGFtcGxlLmNvbTAeFw0yMjEyMTQxNzMxMzda
+Fw0yMzEyMTQxNzMxMzdaMBsxGTAXBgNVBAMMEHB1bHAuZXhhbXBsZS5jb20wggEi
+...
+-----END CERTIFICATE-----
+subject=CN = pulp.example.com
+issuer=CN = pulp.example.com
+```
+
+### **Configuring an existent certificate**
```
Comment (mikedep333) [inline docs/certificates.md]: s/existent/existing.

File: docs/certificates.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+It is also possible to bring your own company certificate instead of creating a new one through [*GENERATING A NEW CERTIFICATE*](#generating-a-new-certificate) steps.
+
+* copy the certificates into `/etc/pulp/certs/` directory. This will prevent having to copy the certificates again in case of a container reprovisioning (the certificates will be persisted in container volume):
+```console
+$ podman cp my-company-generated-certificate.crt pulp:/etc/pulp/certs/pulp_webserver.crt
+$ podman cp my-company-generated-certificate.key pulp:/etc/pulp/certs/pulp_webserver.key
```
Comment (mikedep333) [inline docs/certificates.md:102]: Nice and concise way to do it :)

File: docs/certificates.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+[root@f14649b06e01 /]# cp /etc/pulp/certs/pulp_webserver.crt /etc/pki/tls/certs/pulp_webserver.crt
+cp: overwrite '/etc/pki/tls/certs/pulp_webserver.crt'? y
+[root@f14649b06e01 /]# cp /etc/pulp/certs/pulp_webserver.key /etc/pki/tls/private/pulp_webserver.key
+cp: overwrite '/etc/pki/tls/private/pulp_webserver.key'? y
+```
+
+* restart nginx process to get the new certificate
+```
+$ podman exec pulp s6-svc -r /run/service/nginx
+```
```
Comment (mikedep333) [inline docs/certificates.md:118]: This process is not sufficient. We have  additional commands to run also:
https://github.com/pulp/pulp-oci-images/blob/latest/images/s6_assets/init/certs#L38

**Review by mikedep333 — CHANGES_REQUESTED** [review]

File: docs/certificates.md (hunk 3 above)
Comment (git-hyagi) [reply docs/certificates.md:118]: hum... the addition of the CA into the truststore (which is done by `update-ca-trust` command) is documented in the next section (## SETTING UP ADDITIONAL TRUSTED CAs).
https://github.com/pulp/pulp-oci-images/pull/397/files#diff-b29b72c39bf9c8efd205b812ea4ada8c1a7598b4a04bb632b3345ad39155f462R150
https://github.com/pulp/pulp-oci-images/pull/397/files#diff-b29b72c39bf9c8efd205b812ea4ada8c1a7598b4a04bb632b3345ad39155f462R169


Considering that users could be configuring a certificate signed by a trusted CA (like certsign, letsencrypt, etc) I believe that https://github.com/pulp/pulp-oci-images/blob/latest/images/s6_assets/init/certs#L38 is not a mandatory step, right?


Another thing to notice is that "force-enable" and/or "extract" is/are not needed anymore:
```
$ cat /usr/bin/update-ca-trust
#!/usr/bin/sh

#set -vx

# At this time, while this script is trivial, we ignore any parameters given.
# However, for backwards compatibility reasons, future versions of this script must 
# support the syntax "update-ca-trust extract" trigger the generation of output 
# files in $DEST.
...
```

**Review by mikedep333 — APPROVED** [review]

**Pull request merged**